release.config.js

```diff
@@ -8,8 +8,8 @@ module.exports = {
       prerelease: "${ name }" + `-${process.env.CIRCLE_SHA1}`,
     },
     {
-      name: "renovate/typescript-4.x",
-      prerelease: "ts-4" + `-${process.env.CIRCLE_SHA1}`,
+      name: "renovate/eslint-7.x",
+      prerelease: "eslint-7" + `-${process.env.CIRCLE_SHA1}`,
```

Comment:
uhmmm what is this change about? does renovate do this? @Averethel

Reply:
No, that was me I think. Sometimes semantic release refuses to pick up branches event though they do match our pattern so I went for a manual pre here.